src/inmanta/module.py

```diff
@@ -680,6 +680,12 @@ def _get_module_name(self, module_spec: List[InmantaModuleRequirement]) -> str:
 
 @stable_api
 class ModuleV2Source(ModuleSource["ModuleV2"]):
+    def __init__(self, urls: List[str] = []) -> None:
```

Comment:
Unsafe default here.

Reply:
This is fine as long as the parameter is not mutated right ?

Reply:
Yes, but
1. it should then be typed as `Sequence` so that it would raise a type error if you try
2. I would not rely on it not being mutated. e.g. `pytest-inmanta` does some unorthodox things, it would not surprise me if it toys with this field.
3. I think it's still considered bad practice, though if 1 is met I'm not sure where I personally stand on the issue.

Reply:
I made it sequence.

Reply:
I agree it is not ideal, but this parameter is not in fact used any more. 
It is purely there to maintain api stability.

Reply:
I know, but pytest-inmanta(/-lsm) doesn't use it either then?